src/async-wrap.cc

```diff
@@ -182,6 +184,38 @@ static void Initialize(Local<Object> target,
 }
 
 
+void AsyncWrap::DestroyIdsCb(uv_idle_t* handle) {
+  uv_idle_stop(handle);
+
+  Environment* env = Environment::from_destroy_ids_idle_handle(handle);
+  // None of the V8 calls done outside the HandleScope leak a handle. If this
+  // changes in the future then the SealHandleScope wrapping the uv_run()
+  // will catch this can cause the process to abort.
```

Comment:
heh. seems to have snuck into patches for other versions. I'll remove it here, and remove it in the other branches in an upcoming patch.